ts/session/apis/seed_node_api/SeedNodeAPI.ts

```diff
@@ -230,6 +230,9 @@ async function getSnodesFromSeedUrl(urlObj: URL): Promise<Array<any>> {
 
   const params = {
     active_only: true,
+    // If you are thinking of adding the `limit` field here: don't.
+    // We fetch the full list because when we retrieve it we also remove from all the swarms we already know, any snode not part of that fetched list.
```

Comment:
better?
/**
 * If you are thinking of adding the `limit` field here: don't.
 * We fetch the full list because we will remove from every cached swarms, the snodes not found in that fresh list.
 * If a `limit` was set, we would remove a lot of valid snodes from those cached swarms.
 */

Reply:
All good! Resolved